README.md

```diff
@@ -1 +1,43 @@
-# Rails-Blog-App
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# Rails Blog App
+
+> In this project, I created all controllers and views for my Blog app. I added simple placeholders for my views with plain HTML.
+
+## Built With
+
+- Ruby.
+- Rails.
+- PostgreSQL.
+- Linters.
+
+## Install
+
+- Install Ruby based on your OS from [HERE](https://www.ruby-lang.org/en/downloads/).
+- Install Rails by running this order in your terminal `gem install rails` (good luck with that).
+- Install PostgreSQL from this [link](https://www.postgresql.org/download/).
+
+## Getting started
+
+1. Clone the project using git-bash or Githup Desktop.
+2. Open the project folder with VSCode or any Editor.
+3. Open terminal and navigate to the project folder.
+4. Install all gems by run this command `bundle install`
+5. Open `database.yml` file inside `./config` folder and put your username and password.
+6. Type `rails s` to run live server.
+7. Open your browser on this [127.0.0.1:3000](127.0.0.1:3000)
+8. Now you will see a button saying `create database` click on it and turn off the rails sever an run it again.
```

Comment:
- [ ] **[OPTIONAL]** Good job so far, however, kindly add instructions like run `rails db:create` and `rails db:migrate` to create and migrate the database then run `rails db:seed` to seed data if there's any. This is more explained instructions on how to set up a database.